src/containerapp/azext_containerapp/_params.py

```diff
@@ -365,6 +366,12 @@ def load_arguments(self, _):
         c.argument('environment_name', options_list=['--name', '-n'], help="The environment name.")
         c.argument('yaml', type=file_type, help='Path to a .yaml file with the configuration of a Dapr component. All other parameters will be ignored. For an example, see https://learn.microsoft.com/en-us/azure/container-apps/dapr-overview?tabs=bicep1%2Cyaml#component-schema')
 
+    with self.argument_context('containerapp arc setup-core-dns') as c:
+        c.argument('distro', arg_type=get_enum_type([AKS_AZURE_LOCAL_DISTRO]), help="The distro supported to setup CoreDNS.")
+        c.argument('kube_config', options_list=["--kube-config"], help="Path to the kube config file.")
+        c.argument('kube_context', options_list=["--kube-context"], help="Kube context from current machine.")
+        c.argument('skip_ssl_verification', options_list=["--skip-ssl-verification"], help="Skip SSL verification for any cluster connection.")
```

Comment:
These 3 params not need to set option_list because they are the same with the vars' name. If we want to set multiple options list include a short name, we can set it.